go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/stackitcloud/stackit-cli
 go 1.22
 
 require (
+	github.com/ghodss/yaml v1.0.0
```

Comment:
This one seems to work well, but did you try https://github.com/goccy/go-yaml?

They seem so have around the same number of stars and forks, but https://github.com/ghodss/yaml hasn't been received any update since 2017

Reply:
Changed used library